### DISCUSSION TITLE
Table Width

### DISCUSSION THREAD

**Discussion by laichenkang** [Budibase/budibase#7890] [Help] [answered]
I've only been developing Budibase for a week. So far its been great. One thing I would like to change is the width of the table. This table is autogenerated. I have quite a few columns in the table. On a 1080p screen. The table does not fill the width of the screen. How can I change this? I have ticked Compact. Margins are all None. What else can I change? How can I make the equivalent of Small size.

**Comment by guidot** [comment]
Hi @laichenkang,
You can set the overall width in the Navigation settings:
![grafik](https://user-images.githubusercontent.com/6181359/191721202-93ccede2-533c-4849-b8be-d5a16fd3781c.png)
If you select `Max` your tables will use the whole screen.

**Reply by AlejandroFNadal** [reply]
Thanks a lot, I had not realized this was possible

**Comment by laichenkang** [comment, marked as answer]
I have set Navigation to Max. Only the top part of the screen has stretched to fill the screen. The table below it is still the same.

If it helps, I'm on 1.4.3

Just realized there is another Width to set that is on the right side of the Design page. Setting it to Max works!
Thanks

**Comment by guidot** [comment]
Here it works since at least 1.2.xx (and now I'm on 1.4.7).
In Design view: Does your screen frame show that it is using the whole width?
![grafik](https://user-images.githubusercontent.com/6181359/191744543-318bcc3d-2748-4042-b2ba-10065127768c.png)
You can also see how my screen is configured.

Could you share a screenshot and an export of your app?
